examples/examples_nodejs_test.go

```diff
@@ -44,6 +44,21 @@ func TestUpdate(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestLambdaUpdate(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: filepath.Join(getCwd(t), "lambda-update", "step1"),
+			EditDirs: []integration.EditDir{
+				{
+					Dir:      filepath.Join(getCwd(t), "lambda-update", "step2"),
+					Additive: true,
+				},
+			},
+		})
+
+	integration.ProgramTest(t, &test)
```

Comment:
Does this need some more asserts? Or it used to panic or some such before the change?

Reply:
The update was a hard failure before this change.